python/framework-extensions/langchain/pyproject.toml

```diff
@@ -0,0 +1,59 @@
+[tool.poetry]
+name = "coinbase-agentkit-langchain"
+version = "0.1.0"
+description = "Coinbase AgentKit LangChain extension"
+authors = ["John Peterson <john.peterson@coinbase.com>"]
+readme = "README.md"
+license = "Apache-2.0"
+keywords = ["coinbase", "sdk", "crypto", "cdp", "agentkit", "ai", "agent"]
+packages = [{ include = "langchain" }]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+cdp-sdk = "^0.15.0"
+pydantic = "^2.0"
+web3 = "^7.6.0"
```

Comment:
these are not required. please remove

Reply:
right, i thought they weren't but saw we had them in the TS package